src/TopBar/index.less

```diff
@@ -6,7 +6,8 @@
   min-width: 0;
   transition: all @timingPromptly ease-out;
   .padding--x--2xs;
-  background-image: linear-gradient(90deg, #3980e6 0%, #3965e6 50%, #394de5 100%);
+  /* stylelint-disable-next-line declaration-property-value-whitelist */
+  background-color: #3761e9;
```

Comment:
If this is a new design endorsed color we're going to be using, should we add a variable for it?

Reply:
Good question. I'm going to double-check with design about the specific blue because I don't see this reference anywhere. I'm not sure if this is a new standard color or a one-off.

Reply:
I added a TODO to revisit whether or not to create a color variable.